.github/workflows/rust.yml

```diff
@@ -197,18 +197,17 @@ jobs:
           rust-version: stable
       - name: Generate benchmark data and expected query results
         run: |
-          mkdir -p benchmarks/data/answers
+          mkdir -p datafusion/core/tests/sqllogictests/test_files/tpch/data
           git clone https://github.com/databricks/tpch-dbgen.git
           cd tpch-dbgen
           make
-          ./dbgen -f -s 1
-          mv *.tbl ../benchmarks/data
-          mv ./answers/* ../benchmarks/data/answers/
+          ./dbgen -f -s 0.1
```

Comment:
I looked into this a little more -- and note we are running this CI check sf 0.1 (rather than SF1) It seems like a good idea and means it takes only 8 seconds to make the data 👍 

https://github.com/apache/arrow-datafusion/actions/runs/5108217213/jobs/9181887855?pr=6435